chapters/intro-computer-architecture/c-pointers/demos/pointers/pointers.c

```diff
@@ -0,0 +1,28 @@
+/*
+ * Author: Gabriel Mocanu <gabi.mocanu98@gmail.com>
```

Comment:
is this intended? Shouldn't the license be here?

Reply:
Should we use SPDX format specifiers for license specification? Like Linux kernel does for example?

Reply:
Yup. I will add them tomorrow I think.